RELEASE.md

```diff
@@ -0,0 +1,6 @@
+Release type: patch
+
+Starting with this release, clients using the legacy graphql-ws subprotocol will receive an error when they try to send binary data frames.
```

Comment:
That's a classic Erik: The new subprotocol is called `graphql-transport-ws` and is living in a repository called `graphql-ws`, while the old subprotocol is called `graphql-ws` dying in a repository called `subscriptions-transport-ws`. Throughout our code and docs we refer to the subprotocols name, instead of the repository names.

Reply:
For reference, here's the relevant line in the newer protocols spec: https://github.com/enisdenjo/graphql-ws/blob/c030ed1d5f7e8a552dffbfd46712caf7dfe91a54/PROTOCOL.md?plain=1#L10

The legacy protocol's spec is a mess, but the subprotocol's name (`graphql-ws`) can be found in the reference implementation and various clients supporting it.

Reply:
The title of the document is literally `graphql over ws` and that is the new nomenclature for all the other transport protocols :D lets hope this becomes more consistent in the future

Reply:
Same. This caused way too much confusion over the years :melting_face:

Reply:
Gonna submit an issue let's see if Denis agrees 😂